docs/getting_started.md

```diff
@@ -84,7 +84,77 @@ work for Windows? (If somebody could verify this, that would be great!)
 
 ### OSX
 
-TODO
+NOTE: These commands were tested with Intel MacOS 13.0, they may or may not not work 
+with older versions.
+
+#### Installing the Haskell toolchain
+
+Run the following commands (see https://www.haskell.org/ghcup/) to install `ghcup`, `ghc` and `cabal`. 
+
+
+```bash
+$ curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh
+$ ghcup tui
+# In the terminal UI, select GHC9.0.2, Haskell language server 1.8 and Cabal 3.6.
+# Important: both install + set them! 
+```
+![](https://www.haskell.org/ghcup/ghcup.gif)
```

Comment:
I don't think we should add the gif here, since it's not doing exactly what we are saying?
In the comment we mention what they should do anyway.

```suggestion
```